### PR TITLE
[stable/prisma]: upgrade version to 1.20

### PR DESCRIPTION
**Pull request by giacope**
Signed-off-by: Giacomo Guiulfo <giacomoguiulfo@gmail.com>

#### What this PR does / why we need it:

Upgrades Prisma to latest release.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/prisma/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: prisma
-version: 1.0.0
-appVersion: 1.15
+version: 1.0.1
```
Comment (cpanato) [inline stable/prisma/Chart.yaml]: please bump to `1.1.0`

Comment (giacope) [reply stable/prisma/Chart.yaml]: Fixed

**Comment by cpanato** [comment]
looks like I cannot apply lgtm anymore :/. cc @mattfarina @unguiculus @davidkarlsen

**Pull request merged**